CPPLINT.cfg

```diff
@@ -1,5 +1,9 @@
 set noparent
 
+# When using Velocitas gRPC service generation the header file will include the *ServiceImpl.cpp file
+# this gives by default an error so lets ignore that
+filter=-build/include
```

Comment:
This is needed as cpplint does not like our current approach where we have an include of the *.cpp file at the end of the generated header file.

```
} // namespace velocitas

#include "SeatsServiceImpl.cpp"

#endif // VELOCITAS_SERVICE_IMPL_Seats_H
```